.github/workflows/main.yml

```diff
@@ -0,0 +1,47 @@
+name: Master PR and push
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set Node.js version
+        uses: actions/setup-node@v1
```

Comment:
I think it is safe to use v2 for actions/setup-node@

Reply:
:+1: will try!